advanced/Templates/gravity.db.sql

```diff
@@ -80,13 +80,21 @@ CREATE TABLE gravity
 (
 	domain TEXT PRIMARY KEY
 );
+
+CREATE TABLE auditlist
```

Comment:
Don't add this to this file, create it via the migration script.

Reply:
Why? When we can directly create the correct database, why should we first create an outdated version of the database to then bump it up? I know that we do this in FTL, but this is more because we have a much better error handling in there.

Reply:
That is one of the main points of versioning the database scripts. Once you write the script to create the database at version 1, you don't modify it. If you want to modify the database, you write a new script and upgrade to version 2.

If you modified the first script whenever you made a database update, then you would be duplicating code. Also, there are now more than one possible ways that the database could have gotten to version X, which increases the possibility that there is an error in one of the many ways they could have upgraded.

Reply:
Okay, if you think this is the better way than we can do it like this. Just for the records, I still express my doubts about intentionally having a script that generates an outdated version of a database and then use migration script to jump up one by one to the most recent version.